src/engines/Cesium/hooks/useSceneEvent.ts

```diff
@@ -19,7 +20,9 @@ export function useSceneEvent(type: SceneEventType, callback: SceneEventListener
   useLayoutEffect(() => {
     return scene?.[type].addEventListener((scene: Scene, currentTime: JulianDate) => {
       try {
-        callbackRef.current(scene, currentTime);
+        if (scene.mode !== SceneMode.SCENE2D) {
```

Comment:
Why do we need this?

Reply:
It's a workaround to avoid errors thrown out in 2d mode. Now i removed this workaround and made a bug on task board instead. It's not quite important but maybe need to investigate some in the future.